src/frontend/hooks/server/deviceInfo.ts

```diff
@@ -35,6 +37,17 @@ export const useEditDeviceInfo = () => {
   });
 };
 
+export const useCreatedByToDeviceId = (createdBy: string) => {
+  const project = useActiveProject();
+
+  return useQuery({
+    queryKey: [CREATED_BY_TO_DEVICE_ID_KEY, createdBy],
+    queryFn: async () => {
+      return await project.$createdByToDeviceId(createdBy);
+    },
+  });
+};
```

Comment:
nit: not sure if this belongs here, as device info is more about application-level information about a member's device. this is more about identity within a project, so i'd lean towards placing this in `server/projects`

Reply:
Ok. I am ambivalent about it, so I moved it as you specified.